csrc/scheduler/tools/loop_domain_scheduler.cpp

```diff
@@ -99,6 +99,8 @@ class LoopDomainSchedulerReplayTransform : OptInConstDispatch {
   const std::vector<IterDomain*>& output_ids_;
 };
 
+// Replay a given IterDomain transform expression on the loop domain
```

Comment:
Unrelated to this PR, but this is a comment I had for a past PR but forgot to add.